lib/bandiera/gui/public/site.css

```diff
@@ -25,3 +25,20 @@
 .bandiera-feature-group .panel-body .no-features {
   margin: 15px;
 }
+
+/*bootstarp overrides*/
+@media (min-width:768px){
+  .container {
+    width: 100%;
+  }
+}
+@media (min-width:1200px){
+  .container {
+    width: 90%;
+  }
+}
+@media (min-width:1600px){
+  .container {
+    width: 80%;
```

Comment:
reduced to 80% above 1600px cause otherwise its streched too much and is not comfortable to look at page.
might consider to also add max-width in pixels so it doesnt strech too much on VERY large screens.
(width chosen according to my feeling, not any formal guide...)

i was testing on a 24" screen btw.